data-showcase/src/main/user-interface/src/app/services/data.service.ts

```diff
@@ -198,6 +201,17 @@ export class DataService {
     }
   }
 
+  fetchKeywords(conceptCode: string) {
```

Comment:
Maybe better to move this to `ItemSummaryComponent`, because this data is only locally relevant in that component?
You can then also inject the resourceservice there.